demisto_sdk/commands/pre_commit/pre_commit_command.py

```diff
@@ -186,40 +186,63 @@ def run(
         repos = pre_commit_context._get_repos(pre_commit_context.precommit_template)
         local_repo = repos["local"]
         (
-            docker_hooks,
-            no_docker_hooks,
-        ) = pre_commit_context._get_docker_and_no_docker_hooks(local_repo)
-        local_repo["hooks"] = no_docker_hooks
+            split_hooks,
+            non_split_hooks,
+        ) = pre_commit_context._get_split_and_non_split_hooks(local_repo)
+        logger.info(f'Split hooks: {[hook["id"] for hook in split_hooks]}')
+        logger.info(f'Non-split hooks: {[hook["id"] for hook in non_split_hooks]}')
```

Comment:
```suggestion
        logger.debug(f'Split hooks: {[hook["id"] for hook in split_hooks]}')
        logger.debug(f'Non-split hooks: {[hook["id"] for hook in non_split_hooks]}')
```

will be changed before merging, now its used for debugging issues